src/renderer/widgets/CreateWallet/ui/MultisigWallet/MultisigWalletWizard.tsx

```diff
@@ -0,0 +1,37 @@
+import { useToggle } from '@shared/lib/hooks';
+import { DEFAULT_TRANSITION } from '@shared/lib/utils';
+
+import { MultisigWallet } from './MultisigWallet';
+
+type Props = {
+  isOpen: boolean;
+  onClose: () => void;
+  onComplete: () => void;
+};
+
+export const MultisigWalletWizard = ({ isOpen, onClose, onComplete }: Props) => {
+  const [isModalOpen, toggleIsModalOpen] = useToggle(isOpen);
+
+  const closeMultisigModal = (params?: { complete: boolean }) => {
+    toggleIsModalOpen();
+
+    setTimeout(params?.complete ? onComplete : onClose, DEFAULT_TRANSITION);
+  };
+
+  // const goBack = () => {
```

Comment:
commented code

Reply:
I removed the whole file, it wasn't used.